### PR TITLE
Update Changelog and version info for 1.2.1

### PR DESCRIPTION
**Pull request by dburgener**

### CODE REVIEW

**Comment by dburgener** [comment]
I need to keep notes on the correct way to make a release...  Do I have to push a tag directly to master?  @stevedlawrence I think you set up the release CI.  Can you remind me the right way to do it?

**Comment by stevedlawrence** [comment]
Yep, just push a tag that starts with `v` to master and everything else is automated.

**Comment by stevedlawrence** [comment]
Your changelog changes mention 1.3.0, should be that be 1.2.1?

**Comment by dburgener** [comment]
Do I need to tag a new commit to trigger the CI, or can I just push a tag to the existing HEAD?

File: CHANGELOG [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # SELint Changelog
 
+## [1.3.0] 2022-01-10
```
Comment (stevedlawrence) [inline CHANGELOG:3]: Should be 1.2.1?

Comment (dburgener) [reply CHANGELOG]: fixed, thanks

**Comment by stevedlawrence** [comment]
You just have to push the tag, it's fine if the commit already exists (and probably preferred so you get one last run of the GtiHub actions before officially tagging this). A push of any kind, whether it be a tag or a commit triggers the workflow.

**Comment by dburgener** [comment]
Awesome, thanks.

**Pull request merged**